presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/PlanBuilder.java

```diff
@@ -419,7 +425,9 @@ public TableFinishNode tableDelete(SchemaTableName schemaTableName, PlanNode del
         TableWriterNode.DeleteHandle deleteHandle = new TableWriterNode.DeleteHandle(
                 new TableHandle(
                         new ConnectorId("testConnector"),
-                        new TestingTableHandle()),
+                        new TestingTableHandle(),
+                        TestingTransactionHandle.create(),
+                        Optional.of(TestingHandle.INSTANCE)),
```

Comment:
I am not sure whether the `TableHandle` inside `DeleteHandle` would require a `TableLayoutHandle`. I view `TableLyaoutHandle` as some kind of access path to a table so we can support, e.g. index scan, predicate pushdown, etc. 

Removing a `TableHandle` with non-empty `TableLayoutHandle` sounds a bit off... But I don't have strong opinion here since they are all "TestingLayoutHandle" anyway :) .

Also see the discussion in PrestoSQL about updating TableLaytoutHandle in https://github.com/prestosql/presto/pull/363#discussion_r262007810 :)